docs/man_pages/lib-management/plugin-add.md

```diff
@@ -1,50 +1,68 @@
+Add a specific version of a custom npm or NativeScript module | `$ appbuilder plugin add <Name or ID>@<Version>`
+Add a custom npm or NativeScript module from GitHub URL | `$ appbuilder plugin add <URL>`
+Add a custom npm or NativeScript module from local path | `$ appbuilder plugin add <Path>` 
+<% } %>
 
-Enables a core, integrated or verified plugin for your project. <% if(isHtml) { %>If the plugin has plugin variables and you have not set one or more of them with `--var`, the Telerik AppBuilder CLI shows an interactive prompt to let you set their values.<% } %>
+<% var plugins =""; if(isCordova || isMobileWebsite) { plugins+="Apache Cordova plugins" } if(isHtml || (isConsole && isMobileWebsite)) { plugins+=" or " } if(isNativeScript || isMobileWebsite) { plugins+="custom npm or NativeScript modules" } %>
+
+<% var plugin =""; if(isCordova || isMobileWebsite) { plugin+="Apache Cordova plugin" } if(isHtml || (isConsole && isMobileWebsite)) { plugin+=" or " } if(isNativeScript || isMobileWebsite) { plugin+="custom npm or NativeScript module" } %>
+
+Enables <%=plugins%> for your project. <% if(isHtml) { %>If the Apache Cordova plugin has plugin variables and you have not set one or more of them with `--var`, the Telerik AppBuilder CLI shows an interactive prompt to let you set their values.<% } %>
 <% if(isConsole) { %>
 <% if(isMobileWebsite) { %>
 WARNING: This command is not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help plugin add`
 <% } %>
-<% if(isNativeScript) { %>
-WARNING: This command is not applicable to NativeScript projects. To view the complete help for this command, run `$ appbuilder help plugin add`
 <% } %>
-<% } %>
-<% if((isConsole && isCordova) || isHtml) { %>
+<% if((isConsole && (isCordova || isNativeScript)) || isHtml) { %>
 ### Options
-* `--available` - Lists all plugins that you can enable in your project.
-* `--debug` - Enables the specified plugin for the Debug build configuration only. If `--available` is set, lists all plugins that you can enable for the Debug build configuration.
-* `--release` - Enables the specified plugin for the Release build configuration only. If `--available` is set, lists all plugins that you can enable for the Release build configuration.
-* `--latest` - Enables the latest version of the specified plugin.
-* `--default` - Enables the default version of the specified plugin.
-* `--var.<Variable ID>` - Sets the value for the specified plugin variable in all configurations. If `--debug` or `--release` is specified, sets the variable for the respective configuration only.
-* `--var.debug.<Variable ID>` - Sets the value for the specified plugin variable for the Debug configuration only.
-* `--var.release.<Variable ID>` - Sets the value for the specified plugin variable for the Release configuration only.
+* `--available` - Lists all NativeScript modules that you can enable in your project.
```

Comment:
the if() checks for isCordova. Shoudl we phrase this "Cordova plugins ot NativeScript modules"?